testsuite/MDAnalysisTests/topology/test_pdb.py

```diff
@@ -244,3 +248,26 @@ def test_PDB_metals():
     assert u.atoms[1].mass == pytest.approx(tables.masses["FE"])
     assert u.atoms[2].mass == pytest.approx(tables.masses["CA"])
     assert u.atoms[3].mass == pytest.approx(tables.masses["MG"])
+
+
+def test_PDB_elements():
+    from MDAnalysis.topology import tables
+
+    u = mda.Universe(PDB_elements, format='PDB')
```

Comment:
Do we really not have a PDB file on hand with elements? (I haven't looked.)

Reply:
We do have one with elements (metals.pdb for eg but it contains all hetatoms), but I took @richardjgowers opinion and created an alternative one, which contains an amino acid and some arbitrary metals. I can redirect it to some other pdb if you think that's a better alternative?